test/test_filter_groonga_query_log.rb

```diff
@@ -379,5 +387,104 @@ def emit(configuration, messages)
       event_stream = emit("timezone localtime", messages)
       assert_equal([[@now, statistic]], event_stream.to_a)
     end
+
+    test "datetime_of_time_format" do
```

Comment:
ditto:

```suggestion
    sub_test_case "time_format" do
      test "datetime" do
       # ...
      end
    end
```

I'm not sure whether "datetime" shortcut is needed. Is "datetime" a famous name of this format?